src/AzureExtension/Services/DevBox/DevBoxManagementService.cs

```diff
@@ -120,6 +129,32 @@ public async Task<string> HttpsRequestToDataPlaneWithRawResponse(HttpClient clie
         }
     }
 
+    // Filter out projects that the user does not have write access to
+    private async Task<bool> FilterOutProjectAsync(DevBoxProject project, IDeveloperId developerId)
+    {
+        try
+        {
+            var uri = $"{project.Properties.DevCenterUri}{Constants.Projects}/{project.Name}/users/me/abilities?{Constants.APIVersion}";
+            var result = await HttpsRequestToDataPlane(new Uri(uri), developerId, HttpMethod.Get, null);

```

Comment:
>HttpsRequestToDataPlane [](http://example.com/codeflow?start=31&length=23)

seems like doing a lot of HTTP requests (one for abilities, and one for pools).  they can't both be done in one go?

Reply:
Unfortunately, no. Have to have both.